src/scss/layout/_layout.scss

```diff
@@ -57,35 +43,89 @@ article {
 			display: table;
 		}
 	}
-}
 
-/*
- * Default WP Alignment Classes
- *****************************************************************************/
-figcaption {
-	display: block;
-	font-size: rem(14);
-	font-style: italic;
-	padding: 1rem 0;
-	text-align: center;
+	// WP customized styles for the WeCount website
+
+	// WP styles for vertically aligning text within Media & Text blocks
+	.is-vertically-aligned-center .wp-block-media-text__content {
+		align-items: center;
+		display: flex;
+	}
+
+	.is-vertically-aligned-bottom .wp-block-media-text__content {
```

Comment:
You've defined the vertical alignment for "center" and "bottom" but not for "top". By looking at the computation of [the default value "auto" for `align-self`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self#Specifications), the result depends on the computed value of the parent container. To avoid the unexpected behavior, it's probably better to explicitly specify `align-self` value for `.is-vertically-aligned-top` too.

Reply:
we should support `.has-large-font-size` selector.